qa-pipelines/tasks/cf-deploy-upgrade-common.sh

```diff
@@ -120,6 +120,11 @@ set_helm_params() {
     if [ -n "${KUBE_ORGANIZATION:-}" ]; then
         HELM_PARAMS+=(--set "kube.organization=${KUBE_ORGANIZATION}")
     fi
+    if semver_is_gte 2.11.0 $(helm_chart_version); then
```

Comment:
I don't this has anything to do with version.
the condition can be: if garden_rootfs_driver==overlay-xfs:
                                   then: --set "env.GARDEN_ROOTFS_DRIVER=overlay-xfs"
we don't need `else conditions` as btrfs is default and we wont set garden_rootfs_driver in cap-values for pb or ecp